packages/terra-outline-table-view/src/Table.jsx

```diff
@@ -2,11 +2,58 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import classNamesBind from 'classnames/bind';
+import Table from 'terra-table';
+
+import sectionShape from './proptypes/sectionShape';
+import headerShape from './proptypes/headerShape';
+import widthShape from './proptypes/widthShape';
 import styles from './Table.module.scss';
 
 const cx = classNamesBind.bind(styles);
 
 const propTypes = {
+  /**
+   * An array of sections containing rows.
+   * TODO: Not final shape as row headers and parent/children  heirarchies are still needed to be addressed. NOTE: make sure bodyData rows have toggleAction with a toggleLabel, otherwise tests fail aria
```

Comment:
nit: 
```suggestion
   * TODO: Not final shape as row headers and parent/children  heirarchies are still needed to be addressed. NOTE: make sure bodyData rows have toggleAction with a toggleLabel, otherwise tests fail aria.
```